Source/WebCore/Modules/fetch/FetchBodyOwner.cpp

```diff
@@ -138,47 +137,41 @@ void FetchBodyOwner::blob(Ref<DeferredPromise>&& promise)
         return;
     }
     m_isDisturbed = true;
-    m_body->blob(*this, WTFMove(promise), m_contentType);
+    m_body->blob(*this, WTFMove(promise));
 }
 
 void FetchBodyOwner::cloneBody(FetchBodyOwner& owner)
 {
     m_loadingError = owner.m_loadingError;
-
-    m_contentType = owner.m_contentType;
     if (owner.isBodyNull())
         return;
     m_body = owner.m_body->clone();
 }
 
 ExceptionOr<void> FetchBodyOwner::extractBody(FetchBody::Init&& value)
 {
-    auto result = FetchBody::extract(WTFMove(value), m_contentType);
+    auto currentContentType = contentType();
+    auto contentTypeNotExist = currentContentType.isNull();
```

Comment:
```suggestion
    bool isContentTypeSet = !currentContentType.isNull();
```

Reply:
And then need to change `contentTypeNotExist ` by `!isContentTypeSet` below

Reply:
thanks for the suggestion 🙏